vulcan/_api.py

```diff
@@ -101,73 +99,58 @@ async def _request(
     ) -> Union[dict, list]:
         if self._session.closed:
             raise RuntimeError("The AioHttp session is already closed.")
-
         full_url = (
             url
             if url.startswith("http")
             else self._rest_url + url
             if self._rest_url
             else None
         )
+
         if not full_url:
             raise ValueError("Relative URL specified but no account loaded")
-
         payload = self._build_payload(body) if body and method == "POST" else None
         payload = json.dumps(payload) if payload else None
         headers = self._build_headers(full_url, payload)
-
-        log.debug(" > {} to {}".format(method, full_url))
-
-        # a workaround for aiohttp incorrectly re-encoding the full URL
+        log.debug(f" > {method} to {full_url}")
         full_url = URL(full_url, encoded=True)
-
         async with self._session.request(
             method, full_url, data=payload, headers=headers, **kwargs
         ) as r:
             try:
                 response = await r.json()
                 status = response["Status"]
                 envelope = response["Envelope"]
-
-                # check for the presence of a b64 string preceded with ': '
                 if status["Code"] == 100 and ": " in status["Message"]:
                     raise InvalidSignatureValuesException()
-
                 elif status["Code"] == 108:
-                    log.debug(" ! " + str(status))
+                    log.debug(f" ! {str(status)}")
                     raise UnauthorizedCertificateException()
-
                 elif status["Code"] == 200:
-                    log.debug(" ! " + str(status))
+                    log.debug(f" ! {str(status)}")
                     raise InvalidTokenException()
```

Comment:
These empty lines actually can be removed; it's a switch-case-like construct so the blocks can be close to each other.